lms/djangoapps/discussion_api/api.py

```diff
@@ -276,7 +286,12 @@ def get_thread_list(request, course_key, page, page_size, topic_id_list=None, te
         "per_page": page_size,
         "text": text_search,
     }
+
     text_search_rewrite = None
+
+    if view:
+        query_params[view] = "true"
```

Comment:
Why is "true" stored instead of the actual value of "view" (which from the spec appears that it should be "unread" or "unanswered")?

https://openedx.atlassian.net/wiki/display/MA/Discussion+API+Proposal+Q4+2015